lib/ClangImporter/ImportType.cpp

```diff
@@ -498,6 +501,12 @@ namespace {
       if (pointeeType && pointeeType->isForeignReferenceType())
          return {pointeeType, ImportHint::OtherPointer};
 
+      // If non-copyable generics are disabled, we cannot specify
+      // UnsafePointer<T> with a non-copyable type T.
+      if (pointeeType && pointeeType->isNoncopyable() &&
+          !Impl.SwiftContext.LangOpts.hasFeature(Feature::NoncopyableGenerics))
```

Comment:
@lorentey do you know if this is the right way to check whether we can use `UnsafePointer<NonCopyable>`?